src/components/cms/BrowseSkill/SideDrawer.js

```diff
@@ -13,11 +14,14 @@ const useStyles = makeStyles({
   },
 });
 
-const Button = styled.button`
-  float: right;
+const Button = styled(Fab)`
+  position: fixed;
+  bottom: 20px;
```

Comment:
The button is not completely aligned with chat bubble and scrolltop. The difference was clearly visible when i checked using my UE.
![alignemnt](https://user-images.githubusercontent.com/42002993/74433040-a7bcb300-4e85-11ea-853f-1c82d0d19adb.png)